packages/web-app-files/src/components/SideBar/Details/FileDetails.vue

```diff
@@ -241,7 +255,7 @@ export default defineComponent({
        * webDavPath might not be set when user is navigating on public link,
        * even if the user is authenticated and the file owner.
        */
-      return resourcesStore.areWebDavDetailsShown && unref(resource).webDavPath
+      return resourcesStore.areWebDavDetailsShown && unref(resource)?.webDavPath
```

Comment:
```suggestion
      return resourcesStore.areWebDavDetailsShown && unref(resource).webDavPath
```
No need for that check